src/LibChorusTests/VcsDrivers/Mercurial/RepositoryTests.cs

```diff
@@ -165,10 +165,18 @@ public void CloneToUsbWithoutUpdateFollowedByIdentifierDoesNotAffectHgrc()
 
 		private static string GetExtensionsSection(string pathToMercurialFolder)
 		{
+#if !MONO
```

Comment:
I'm alarmed here. So the two platforms should have the same extensions. The data can change (for path purposes) but I don't think we can have users on different OS work together if we can't have the same extensions.